Sources/Mantis/Helpers/TransformStack.swift

```diff
@@ -0,0 +1,59 @@
+import UIKit
+
+
+class TransformStack: NSObject {
```

Comment:
Since the class is not public now, you can remove` public` from its properties now.

Reply:
changes made.

Reply:
I can still see public properties and functions in this class, please check if they are necessary.

Reply:
changes made.